packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -406,9 +406,7 @@ class AnimationController extends Animation<double>
     } else if (_value == upperBound) {
       _status = AnimationStatus.completed;
     } else {
-      _status = (_direction == _AnimationDirection.forward) ?
-        AnimationStatus.forward :
-        AnimationStatus.reverse;
+      _status = (_direction == _AnimationDirection.forward) ? AnimationStatus.forward : AnimationStatus.reverse;
```

Comment:
For simple cases I prefer each branch of the ternary on its own line

Reply:
You can add a trailing comment to get the formating you prefer:

```dart
  _status = (_direction == _AnimationDirection.forward) //
      ? AnimationStatus.forward
      : AnimationStatus.reverse;
```